### ISSUE TITLE
Notifications doesn't work with snap release

### ISSUE THREAD

**Issue by keypush3r** [squidowl/halloy#501]
On ubuntu 22.04 the 

```
[notifications.highlight]
show_toast = true
sound = "dong"
``` 
Doesn't do anything while running the snap release (24.10). Worth noting that I receive this error in the syslog:

```
<time> <host> halloy_halloy.desktop[84869]: ALSA lib pcm.c:2664:(snd_pcm_open_noupdate) Unknown PCM default 
```

...but even if I remove the sound = "dong" directive, there is no desktop notification. The sound part *can* be an issue with my system but aren't having trouble with other applications.

**Comment by qauff** [comment]
I'm not sure how it's done with snap (I never figured it out) but with flatpak, you need to give audio permissions to halloy. There may be something similar with snaps.

**Comment by casperstorm** [comment]
This should have been fixed in latest version (2024.12).
